WebApp/static/oidc/oidc-client/dist/oidc-client.min.js

```diff
@@ -1,46 +1,15428 @@
-/*!
+var Oidc = (function (t) {
```

Comment:
I'm not sure it should be stylized and all the minified versions like as `*.min.js`